web/static/js/redux/alert.js

```diff
@@ -1,5 +1,7 @@
 const types = {
   CLEAR_ALERT: "CLEAR_ALERT",
+  RETRO_UPDATE_COMMITTED: "RETRO_UPDATE_COMMITTED",
```

Comment:
I realize this is a refactor, but if we're going to clean this up, would it be better to import the `types` from the "retro" redux file?

Reply:
and in fact, `SHOW_STAGE_HELP` is also in the `retros` redux file's `types`, so if we *can* import them, I think we should house both there. any reason we wouldn't? not sure if there's a redux convention for this

Reply:
Hmm - convention is all over the place here.  Since we're housing reducers and actionCreators together, we should probably split the types.  So SHOW_STAGE_HELP and CLEAR_ALERT stay in the alert redux file, and the others stay in the retros file.  We then import in both files...